dist/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>htmlWebpackPlugin.options.title</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script defer src="bundle990b8dbce86cdc9201a5.js"></script>
+<script defer src="bundle9fc3b336b0b77e65d2d8.js"></script></head>
+
+<body>
+  <main>
+    <section class="container">
+      <p class="notification hide">Form submitted</p>
+
+      <form>
+        <h1>Enter your Score</h1>
+        <input type="text" placeholder="Enter your name" class='nameInput' required>
+        <input type="text" placeholder="Enter your score" class='scoreInput' required>
```

Comment:
- [ ] [REQUIRED] Please change your input Score to `type="text" => type="number"` ! Becouse it is not best HTML practice.

![image](https://user-images.githubusercontent.com/63915024/182872442-38154c0d-04a3-469d-835e-2061a619b048.png)